phpseclib/Crypt/Rijndael.php

```diff
@@ -755,7 +755,7 @@ function _subWord($word)
     {
         static $sbox;
         if (empty($sbox)) {
-            list(,,,, $sbox) = $this->_getTables();
+            list(, , , , $sbox) = $this->_getTables();
```

Comment:
Ahhh, hmmmm, this issue came up in Laravel the other day, which is why I suggested it. I'll let Taylor know.